pkg/versioninfo/variables.go

```diff
@@ -15,30 +15,12 @@
 package versioninfo
 
 import (
-	_ "embed"
-	"strings"
 	"time"
 )
 
-// @TODO
-// The placement of version.txt inside this package is not ideal.
-// Unfortunately Go does not allow ".." in embed paths. A possible workaround is to
-// use "go:generate" comment to copy the file from the root to this location (and adding
-// the local version to .gitignore...).
-
-// An alternative to using a fixed file, would be to use a go:generate comment with
```

Comment:
question (curiosity): Did you run into issues using `go:generate` for this, or was `LD_FLAGS` better/easier?

Reply:
I didn't want to do 2 operations of go build and go generate that's why I used LD_FLAGS.